misc/language.c

```diff
@@ -21,7 +21,7 @@
 #include <stdint.h>
 
 #include "common/common.h"
-#include "misc/bstr.h"
+#include "misc/ctype.h"
```

Comment:
why remove bstr.h?

Reply:
It's moved to misc/language.h so the prototype can have `bstr` instead of `struct bstr`.